testcafe/README.md

```diff
@@ -0,0 +1,24 @@
+# Cozy TestCafe
+
+[Testcafe](https://devexpress.github.io/testcafe/) is used for automated testing on Travis.
+
+To run local test using testcafe, you must first set some environment variables.
+
+```console
+export TESTCAFE_PHOTOS_URL="your Cozy Photos-url" #ie: https://username-photos.mycozy.cloud/
+export TESTCAFE_USER_PASSWORD="your password"
+```
+
+Then, in your git folder, use node to run our tests set :
+
+```console
+node testcafe/runner_photos.js
```

Comment:
I would recommend we document the `yarn photos` command instead, as it will not change. `runner_photos.js` might end up renamed or moved, and we'll definitely forget to update the documentation 😛